src/eureka/S5_lightcurve_fitting/models/QuasiLambertianPhaseCurve.py

```diff
@@ -95,7 +95,8 @@ def eval(self, channel=None, pid=None, **kwargs):
                     phi = anom + pl_params.w*np.pi/180 + np.pi/2
 
                 # calculate the phase variations
-                phaseVars = np.abs(np.cos((phi-self.quasi_offset*np.pi/180)/2)
+                phaseVars = np.abs(np.cos(
+                                   (phi+pl_params.quasi_offset*np.pi/180)/2)
```

Comment:
This change is also needed in the `differentiable_models` version of the function

Reply:
One day, I will remember to change things in both versions.

Reply:
One day, I hope that we can merge these two versions of the code (potentially aided by switching from PyMC3 to jax) so that we don't have to duplicate our efforts